amgut/static/js/american_gut.js

```diff
@@ -1,3 +1,13 @@
+  function getParameterByName(name, url) {
```

Comment:
This function is way too obfuscated as it is, would you mind adding some explanation as to what's happening here?